packages/atomic/project.json

```
@@ -38,46 +15,48 @@
           "node ./scripts/process-css.mjs --config=tsconfig.lit.json ",
           "npm run-script -w=@coveo/release if-not-cdn -- \"rollup -c rollup.config.js\"",
           "esbuild src/autoloader/index.ts --format=esm --outfile=dist/atomic/autoloader/index.esm.js",
-          "esbuild src/autoloader/index.ts --format=cjs --outfile=dist/atomic/autoloader/index.cjs.js"
+          "esbuild src/autoloader/index.ts --format=cjs --outfile=dist/atomic/autoloader/index.cjs.js",
+          "rimraf ./dist/atomic/loader/package.json",
+          "tsc --noEmit --esModuleInterop --skipLibCheck ./dist/types/components.d.ts"
         ],
         "parallel": false,
-        "cwd": "packages/atomic"
+        "cwd": "{projectRoot}"
       }
     },
-    "validate:definitions": {
-      "dependsOn": ["^build", "cached:build:stencil"],
-      "inputs": [],
-      "outputs": [],
-      "executor": "nx:run-script",
+    "build-storybook": {
+      "dependsOn": ["build-stencil-lit", "cem:build", "list-assets"],
```

```suggestion
      "dependsOn": ["cem:build", "list-assets"],
```

`build-stencil-lit` is superfluous here since it's already in the `dependsOn` of `list-assets`